docs/gallery/domain/plot_behavior.py

```diff
@@ -0,0 +1,415 @@
+#
+# For position data ``reference_frame`` indicates the zero-position, e.g.
+# the 0,0 point might be the bottom-left corner of an enclosure, as viewed from the tracking camera.
+
+timestamps = np.linspace(0, 50) / 200
+
+position_spatial_series = SpatialSeries(
+    name="SpatialSeries",
+    description="(x,y) position in open field",
+    data=position_data,
+    timestamps=timestamps,
+    reference_frame="(0,0) is bottom left corner",
+)
+
+position_spatial_series
+
+####################
+#
+# .. seealso::
+#    You can learn more about best practices that can be applied to
+#    :py:class:`~pynwb.behavior.SpatialSeries` `here <https://www.nwb.org/best-practices/#timeseries>`_.
+#
+# To help data analysis and visualization tools know that this :py:class:`~pynwb.behavior.SpatialSeries` object
+# represents the position of the subject, store the :py:class:`~pynwb.behavior.SpatialSeries` object inside
+# of a :py:class:`~pynwb.behavior.Position` object, which can hold one or more :py:class:`~pynwb.behavior.SpatialSeries` objects.
+
+position = Position(spatial_series=position_spatial_series)
+
+
+####################
+# Create a Behavior Processing Module
+# --------------------------
+#
+# .. seealso::
+#    You can read more about the basic concepts of processing modules in the :ref:`modules_overview` tutorial.
+#
+# Create a processing module called ``"behavior"`` for storing behavioral data in the :py:class:`~pynwb.file.NWBFile`
+# and add the :py:class:`~pynwb.behavior.Position` object to the processing module using the
+# :py:meth:`~pynwb.file.NWBFile.create_processing_module` method:
+
+
+behavior_module = nwbfile.create_processing_module(
+    name="behavior", description="processed behavioral data"
+)
+
+behavior_module.add(position)
+
+
+####################
+# CompassDirection
+# ------------
+#
+# Spatial Series
+# ^^^^^^^^^^^^^^^
+#
+# Analogous to how position can be stored, we can create a :py:class:`~pynwb.behavior.SpatialSeries`
+# object for representing the view angle of the subject.
+#
+# For direction data ``reference_frame`` indicates the zero-axes, for instance
+# “straight-ahead” might be a specific pixel on the monitor, or some other point in space.
+# The unit of measurement for the :py:class:`~pynwb.behavior.SpatialSeries` object should be radians or degrees.
+
+view_angle_data = np.linspace(0, 4, 50)
+
+direction_spatial_series = SpatialSeries(
+    name="SpatialSeries",
+    description="view angle",
+    data=view_angle_data,
+    timestamps=timestamps,
+    reference_frame="bottom left",
+    unit="radians",
+)
+
+direction = CompassDirection(
+    spatial_series=direction_spatial_series, name="CompassDirection"
+)
+
+####################
+# We can add a :py:class:`~pynwb.behavior.CompassDirection` object to the ``behavior`` processing module
+# the same way as we have added the position data:
+
+behavior_module.add(direction)
+
+
+####################
+# BehavioralTimeSeries
+# ------------
```

Comment:
Just a general thought, I think it would be nice to have the heading indicate not just the name of the neurodata_type, but what the section is about, e.g., **BehavioralTimeSeries: Storing continuous behavior** to make it easier for readers to find relevant sections. I know we are not doing this right now in the other tutorials either, but I think it would be useful. This is not critical and it's fine if you decide to leave the headings as is.